assets/js/lib/test-utils/factories/users.js

```diff
@@ -5,12 +5,12 @@ import { formatISO } from 'date-fns';
 export const userFactory = Factory.define(() => ({
   id: faker.number.int(),
   username: faker.internet.userName(),
-  created_at: formatISO(faker.date.past()),
   actions: 'Delete',
   enabled: faker.datatype.boolean(),
   fullname: faker.internet.displayName(),
   email: faker.internet.email(),
-  password: faker.internet.password(),
```

Comment:
At the end, we don't need it.
The password is never returned by our user api's, because it is a sensitive information.
for that, we don't really need in the factory

Reply:
Yeah, makes sense, it was just handy for tests, but you are fully right, we should stick with the backend truth :)